gamedata/alldefs_post.lua

```diff
@@ -238,8 +238,41 @@ function UnitDef_Post(name, uDef)
 			end
 		end
 
+		--normal commander respawning
+		if modOptions.comrespawn == "all" then
+			if name == "armcom" or name == "corcom" or name == "legcom" then
```

Comment:
Why not `if uDef.customparams.iscommander then`?

Reply:
Because many things that aren't commanders have iscommander. It's silly.

Reply:
Are you talking about `armcomcon` and `corcomcon`? Those are the only ones I see that could be considered "Commanders that aren't Commanders"

Reply:
Yep. Them be the ones.

Reply:
Oh, another good reason is that some tweakdefs give iscommander to non-commander units. This would cause unintended interactions.